sample-app/src/components/FilterSearch.tsx

```diff
@@ -0,0 +1,124 @@
+import { useState } from "react";
+import { useAnswersActions, FilterSearchResponse, SearchParameterField } from '@yext/answers-headless-react';
+import InputDropdown from "./InputDropdown";
+import renderWithHighlighting from "./utils/renderWithHighlighting";
+import DropdownSection, { Option } from "./DropdownSection";
+import { processTranslation } from "./utils/processTranslation";
+import { useSynchronizedRequest } from "../hooks/useSynchronizedRequest";
+
+const SCREENREADER_INSTRUCTIONS = 'When autocomplete results are available, use up and down arrows to review and enter to select.'
+
+export interface FilterSearchProps {
+  title: string,
+  sectioned: boolean,
+  searchFields: Omit<SearchParameterField, 'fetchEntities'>[],
+  screenReaderInstructionsId: string,
+  customCssClasses?: {
```

Comment:
We should follow the example in this file: https://github.com/yext/answers-headless-react/pull/65/files#diff-19c78cfd82fdc3cf822987838108d3efc87b37b12796ddffaecda9a58544dc00. Basically, you'll define an interface for the css classes of this Component, say, `FilterSearchCssClasses`. Then, here, you'll have `customCssClasses?: FilterSearchCssClasses`. 

We'll want to do a similar thing for `InputDropdown` and the Section css classes too.

Reply:
Updated to have an interface for each component and moved the css classes currently used for styling from `VerticalSearchPage` into `FilterSearch` as the built-in css classes